src/optim_problems/opt_cep.jl

```diff
@@ -82,7 +83,11 @@ function setup_opt_cep_basic_variables!(cep::OptModelCEP,
   @variable(cep.model, COST[account=set["account"],impact=set["impact"],tech=set["tech"]])
   # Capacity
   push!(cep.info,"Variable CAP[tech, infrastruct, nodes] ≥ 0 in MW]")
+  if "tech_transmission" in keys(set) #if transmission is in tech, exclude the transmission technology from the setup of variable CAP
```

Comment:
For case of transmission being in tech of data, it is more elegant to not setup CAP["trans"]=0, but not set it up entirely